packages/s-deploy/src/index.ts

```diff
@@ -7,13 +7,15 @@ const getPluginInputs = async (inputs: Record<string, any>, context: Record<stri
   logger.debug(`original inputs: ${JSON.stringify(inputs)}`);
   const newInputs = _.assign(getInputs(inputs, context) as Record<string, any>, {
     deployFile: _.get(context, 'inputs.context.data.deployFile'),
-    envName: _.get(context, 'inputs.context.data.envDeploymentName')
+    envName: _.get(context, 'inputs.context.data.envName'),
```

Comment:
`envDeploymentName -> envName` could be a breaking change. 
Please make sure no one has used this plugin before.